CODEOWNERS

```diff
@@ -295,6 +296,7 @@ Kconfig*                                  @tejlmand
 /tests/lib/pcm_mix/                       @koffes @alexsven @erikrobstad @rick1082 @gWacey
 /tests/lib/pcm_stream_channel_modifier/   @koffes @alexsven @erikrobstad @rick1082 @gWacey
 /tests/lib/tone/                          @koffes @alexsven @erikrobstad @rick1082 @gWacey
+/tests/lib/sample_rate_converter/         @andvib @gWacey
```

Comment:
please keep the alphabetical order of the modules and tests

Reply:
I'm not exactly sure how to follow the structure as most entries aren't alphabetical, but at least I moved the entry above `tone` so that it's sorted within the Octave libs